test/unit/query-generator/select-query.test.ts

```diff
@@ -47,6 +53,27 @@ describe('QueryGenerator#selectQuery', () => {
     });
   });
 
+  it('supports querying for bigint values', () => {
+    const sql = queryGenerator.selectQuery(Project.tableName, {
+      model: Project,
+      attributes: ['id'],
+      where: {
+        duration: { [Op.eq]: BigInt('9007199254740993') },
```

Comment:
```suggestion
        duration: { [Op.eq]: 9007199254740993n },
```

Reply:
I pulled in this suggestion, but it makes the build fail with `ERROR: Big integer literals are not available in the configured target environment ("node10")`: https://github.com/sequelize/sequelize/runs/6429489596?check_suite_focus=true#step:6:14

Since sequelize 7 [targets node.js 14.17.0+](https://sequelize.org/docs/v7/other-topics/upgrade/#minimum-supported-engine-versions) I've updated the esbuild config to target `node14` on my branch: 1a5dcb102

Let me know if I should make a separate PR with that change, or dial back to the `BigInt('9007199254740993')` syntax instead.

Reply:
Looks like we missed that when we dropped node 12, it's fine to do it in this PR :)